### DISCUSSION TITLE
Copilot free plan for non-profit projects

### DISCUSSION THREAD

**Discussion by edgarriba** [community/community#44295] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi, I’m leading the [Kornia](https://github.com/kornia/kornia) project, which is registered as a non-profit org.
We were wondering whether you offer free plans to use copilot for the core maintainers of such projects.

I tried from my personal account but got a message saying that the extended access has ended.

**Comment by iamcjwilliams** [comment, marked as answer]
@edgarriba Yes, The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted :)

**Reply by edgarriba** [reply]
Thanks so, I’ll do!

**Reply by vladutilie** [reply]
Hi, @edgarriba! Have you got an answer? Any chance to get GitHub Copilot for NGOs for free? Thank you!

**Reply by sripwoud** [reply]
Same here, I would be interested to know if that is possible.

**Reply by Codaqua** [reply]
Same here! any news?

**Reply by sripwoud** [reply]
Answer I got after opening a Support ticket:
> We currently do not offer any discounts for GitHub Copilot.

![image](https://user-images.githubusercontent.com/38692952/224556270-06671d45-fd2a-45f5-af20-8ad5647b1f6e.png)

**Reply by edgarriba** [reply]
@r1oga thanks! and so sad 🥲

**Reply by valentinkravtchenko** [reply]
Same here, it would have been very valuable for a non-profit:S

**Reply by irakli-research** [reply]
Same, it would help my non-profit organization as well.

**Reply by genidma** [reply]
Maybe a not-for-profit can be registered via an accredited institution like Stripe. Once approved, Github can then offer discounted rates for not-for-profits and other public-benefit-organizations. One idea. Thx

**Reply by wtvamp** [reply]
Techsoup would be good for something like this: https://www.techsoup.org/

**Reply by greggdavenport** [reply]
Hear hear, through TechSoup would be huge for our nonprofit too!

**Reply by jalbaiges** [reply]
Any news on this? It would be great that _Github for non-profits_ programme included something about Copilot. It also would be great to have any option via Techsoup, as well. Thanks for hearing!

cc @iamcjwilliams

**Reply by valentinkravtchenko** [reply]
we need a canadian version techsoup.ca
